chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -293,7 +293,7 @@ public UserInvitation inviteUser(@RequestBody final UserInvitation invitation)
                 + env.getProperty("chplUrlBegin") + "/#/registration/create-user/" + createdInvite.getInviteToken()
                 + "</p>"
                 + "<p>If you have any issues completing the registration, "
-                + "please visit the <a href=\"https://inquiry.healthit.gov/support/\">Health IT Feedback and Inquiry Portal</a> and select \"Certified Health IT Product List (CHPL)\" to submit a ticket.</p>"
+                + "please visit the <a href=\"https://inquiry.healthit.gov/\">Health IT Feedback and Inquiry Portal</a> and select \"Certified Health IT Product List (CHPL)\" to submit a ticket.</p>"
```

Comment:
I supppose you don't technically need the last back slash and escape character anymore - but also, doesn't really matter either way - still approving